CHANGELOG.md

```diff
@@ -67,7 +67,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * (core/types) [\#6794](https://github.com/cosmos/ibc-go/pull/6794) The composite interface `QueryServer` has been removed from package `core/types`. Please use the granular `QueryServer` interfaces provided by each core submodule.
 * (light-clients/06-solomachine) [\#6888](https://github.com/cosmos/ibc-go/pull/6888) Remove `TypeClientMisbehaviour` constant and the `Type` method on `Misbehaviour`.
 * (light-clients/06-solomachine, light-clients/07-tendermint) [\#6891](https://github.com/cosmos/ibc-go/pull/6891) The `VerifyMembership` and `VerifyNonMembership` functions of solomachine's `ClientState` have been made private. The `VerifyMembership`, `VerifyNonMembership`, `GetTimestampAtHeight`, `Status` and `Initialize` functions of tendermint's `ClientState` have been made private.
-
+ * (core/02-client) [\6934](https://github.com/cosmos/ibc-go/pull/6934) The `GetRouter` method has been removed from the keeper. `AddRoute` method has been added as a wrapper that replaces `GetRouter().AddRoute()`
```

Comment:
We don't need the changelog because the `GetRouter` function has not been released yet.